apps/web/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "build": "next build",
-    "db:push": "dotenv drizzle-kit push:pg",
+    "db:push": "dotenv drizzle-kit push",
```

Comment:
it doesn't need to specify the database no more?

Reply:
Yes, in the new `drizzle` version they moved it to the config.